bevm/bevmclient/README.md

```diff
@@ -0,0 +1,40 @@
+Navigation: [DEDIS](https://github.com/dedis/doc/tree/master/README.md) ::
+[Cothority](../README.md) ::
+[Building Blocks](../doc/BuildingBlocks.md) ::
+[BEvm](https://github.com/dedis/cothority/blob/master/bevm/README.md) ::
+bevmclient
+
+# bevmclient - CLI tool to deploy and interact with BEvm contracts
+
+For the details on all the options and arguments, invoke the tool using the `--help` option.
+
+## Create a BEvm account
+Assuming ByzCoin config and key files in the current directory (see [bcadmin](https://github.com/dedis/cothority/blob/master/byzcoin/bcadmin/README.md) for details):
+```bash
+bevmclient create_account --account-name <MyAccount>
+```
+
+## Credit a BEvm account
```

Comment:
I am not sure I understand what you mean by crediting an account. Is this for putting gas? In any case I think it should be depositing, not crediting.

Reply:
It is for putting Ether in an Ethereum account.
I used crediting as in "add (an amount of money) to an account" (credit vs debit). You find it confusing?